docs/dev/README.md

```diff
@@ -1,35 +1,52 @@
 
-The initial goal is to generate some code that collects the desired metrics in a reliable way, and to ensure that the general integration framework is in place. Start by writing the basic functionality as a custom Check, then fill in the framework details from [Create a New Integration][2].
+### [Agent-based Integration][2]
+Agent-based integrations use the Datadog Agent to submit data via checks written by the partner. The implementation code for these integrations is hosted by Datadog. Agent integrations are best suited for collecting data from custom systems or applications. Writing an agent integration requires you to publish and deploy your solution as a Python wheel (.whl).
+
+## Why create an integration?
 
-Next, open a pull request against the [integrations-extras repository][3]. This signals to Datadog that you're ready to start reviewing code together. Don't worry if you have questions about tests, Datadog internals, or other topics - the integrations team is ready to help, and the pull request is a good place to go over those concerns. Be sure to take advantage of the [Community Office Hours][4] as well!
+Out-of-the-box Metrics - Metrics reported from official Datadog integrations (unless the integration is sending in potentially unlimited metrics) are not counted as custom metrics, and therefore won't impact a customer's custom metric allocation.
 
-Once the integration has been validated (functionality, framework compliance, and general code quality) it is merged into Extras. Once there, it becomes part of the Datadog ecosystem. Congratulations!
+Adoption - Ensuring native support for Datadog reduces friction to adoption and incentivizes Datadog customers to build out their technology stack with partner technologies.
 
-### What are your responsibilities?
+Added Visibility - Partners integrations appear on the Integrations page alongside all Datadog-built integrations, providing key visibility to all of Datadog's customers.
 
-Going forward, you, as the author of the code, are the active maintainer of the integration. You're responsible for maintaining the code and ensuring the integration's functionality. There is no specific time commitment, but you must be a maintainer and take care of the code for the foreseeable future. Datadog extends support on a best-effort basis for Extras, so you are not alone.
+### Responsibilities
+Going forward, you, as the author of the integration, are the active maintainer of the integration. You are responsible for maintaining the code and ensuring the integration’s functionality. There is no specific time commitment, but you must be a maintainer of the code for the foreseeable future. Datadog extends support on a best-effort basis for partner-built integrations, so please reach out to Datadog support if help is needed.
 
-## Let's get started!
+## Let's Get Started 
+To create an API integration, [click here][1].
+To create an Agent-based integration, [click here][2].
```

Comment:
Are these on new lines? It's on the same line in the preview but it would be easier to read if it's on a new line.